src/main/java/com/dreammaster/gthandler/recipes/VacuumFurnaceRecipes.java

```diff
@@ -23,41 +24,43 @@ public class VacuumFurnaceRecipes implements Runnable {
     @Override
     public void run() {
 
-        GTValues.RA.stdBuilder()
-                .itemInputs(
-                        GTUtility.getIntegratedCircuit(12),
-                        GTBees.combs.getStackForType(CombType.INDIUM, 64),
-                        GTBees.combs.getStackForType(CombType.INDIUM, 64),
-                        GTBees.combs.getStackForType(CombType.INDIUM, 64),
-                        GTBees.combs.getStackForType(CombType.INDIUM, 64))
-                .itemOutputs(
-                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Zinc, 64L),
-                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Zinc, 64L),
-                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Zinc, 52L),
-                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Iron, 64L),
-                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Iron, 56L),
-                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Indium, 64L),
-                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Indium, 64L),
-                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Indium, 64L),
-                        MaterialsElements.getInstance().GERMANIUM.getDust(15))
-                .fluidInputs(FluidUtils.getFluidStack(SphaleriteFlotationFroth, 4000))
-                .fluidOutputs(FluidUtils.getFluidStack(AgriculturalChem.RedMud, 2000), FluidUtils.getWater(2000))
-                .eut((int) TierEU.RECIPE_LuV).metadata(COIL_HEAT, 5500).duration(2 * MINUTES)
-                .addTo(vacuumFurnaceRecipes);
+        if (Forestry.isModLoaded()) {
```

Comment:
if this removes the while content of the file, maybe put the mod check where this class is instanciated ?

Reply:
no. people should feel free to add more recipes to the file that dont dep on forestry.

Reply:
ok :)